core/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -490,6 +551,16 @@ public void updateSettings(Settings settings) {
         }
     }
 
+    private static void validate(Map<String, Settings> groupSettings) {
+        for (String key : groupSettings.keySet()) {
+            if (!THREAD_POOL_TYPES.containsKey(key)) continue;
```

Comment:
Can you expand this to the full form, it's too easy to accidentally lose it without the `{}`